demo-app/client/generated/store/career/positions/states/effects.ts

```diff
@@ -9,16 +9,17 @@ import {HttpErrorResponse} from '@angular/common/http';
 import {Injectable} from '@angular/core';
 import {Actions, Effect} from '@ngrx/effects';
 
+import {Observable} from 'rxjs/Observable';
 import {of} from 'rxjs/observable/of';
-
 import {catchError, map, switchMap} from 'rxjs/operators';
+
 import {CareerService} from '../../../../controllers/Career';
 import * as actions from './actions';
 
 @Injectable()
 export class PositionsEffects {
   @Effect()
-  Positions = this.storeActions.ofType<actions.Start>(actions.Actions.START).pipe(
+  Positions: Observable<actions.PositionsAction> = this.storeActions.ofType<actions.Start>(actions.Actions.START).pipe(
```

Comment:
@katemihalikova, please, could you explain why do we need this?
Originally, `Observable<actions.Success | actions.Error>` was inferred. What does this explicit typing bring? Thx.